src/components/Create/components/pages/NftRewards/NftRewardsPage.tsx

```diff
@@ -57,210 +61,218 @@ export const NftRewardsPage = () => {
             <RewardsList allowCreate />
           </Form.Item>
 
-          <Space
-            className="w-full pt-3 pb-2"
-            direction="vertical"
-            size="middle"
-          >
-            <Form.Item
-              name="collectionName"
-              label={
-                <TooltipLabel
-                  label={t`Collection Name`}
-                  tip={
-                    <Trans>
-                      Your collection's name is used on NFT marketplaces (for
-                      example, OpenSea).
-                    </Trans>
-                  }
-                />
-              }
-              required
-            >
-              <JuiceInput />
-            </Form.Item>
-
-            <Form.Item
-              name="collectionSymbol"
-              label={
-                <TooltipLabel
-                  label={t`Collection Symbol`}
-                  tip={
-                    <Trans>
-                      Your collection's symbol is used on NFT marketplaces (for
-                      example, OpenSea).
-                    </Trans>
-                  }
-                />
-              }
-              required
-            >
-              <JuiceInput />
-            </Form.Item>
-
-            <Form.Item
-              name="collectionDescription"
-              label={<Trans>Collection Description</Trans>}
+          {hasNfts && (
```

Comment:
None of this code changed, just nested due to this line here to toggle hidden or not